README.md

```diff
@@ -218,9 +219,14 @@ Thank you to all the people who already contributed Armbian!
 ## Armbian Partners
 
 Armbian's partnership program helps to support Armbian and the Armbian community! Please take a moment to familiarize yourself with our Partners:
+
 - [Click here to visit our Partners page!](https://armbian.com/partners)
 - [How can I become a Partner?](https://forum.armbian.com/subscriptions)
 
+## Star History
+
+[![Star History Chart](https://api.star-history.com/svg?repos=armbian/build&type=Date)](https://star-history.com/#armbian/build&Date)
```

Comment:
A gimmick, for sure... can't hurt?

Reply:
shows something where we are currently good at ;)